core/components/molecules/inputMask/InputMask.tsx

```diff
@@ -22,6 +22,11 @@ export interface MaskProps extends BaseProps {
    * Adds caption to `input` on error
    */
   caption?: string;
+  /**
+   * custom Validator for `InputMask`
+   * `boolean | ((val?: string) => boolean)`
+   */
+  validator?: boolean | ((val: string) => boolean);
```

Comment:
Validator should be type of function only.
`validator?: ((val: string) => boolean);`